tools/cli/commands/docker.js

```diff
@@ -624,6 +626,18 @@ export function dockerDefine( yargs ) {
 					builder: yargExec => defaultOpts( yargExec ),
 					handler: argv => execDockerCmdHandler( argv ),
 				} )
+				.command( {
+					command: 'select-php <version>',
+					description:
+						'Select the version of PHP for use inside the container. See documentation for important notes!',
+					builder: yargCmd => {
+						yargCmd.positional( 'version', {
+							describe: 'The version to select, or "default".',
```

Comment:
Hmm, can we maybe put a list of supported version numbers in there? I personally find little things like that quite helpful, even if it's just to check the expected format. It's a small list anyway.

Adding some basic validation on that would be cool too.

Reply:
Not here without manually hard-coding them. It depends on which versions are available from https://deb.sury.org/.

I didn't bother with validation here since the exec'ed command itself does what would be the same validation.